packages/components/src/components/shared/FormText.test.tsx

```diff
@@ -14,24 +14,49 @@ jest.mock('react-native/Libraries/Utilities/useColorScheme', () => {
 })
 
 describe('Form Text', () => {
-  const header = { text: 'Header text', a11yLabel: 'Header a11y' }
-  const hint = { text: 'Hint text', a11yLabel: 'Hint a11y' }
-  const description = {
-    text: 'Description text',
+  const headerString = 'Header string'
+  const headerObject = { text: 'Header text object', a11yLabel: 'Header a11y' }
+
+  const hintString = 'Hint string'
+  const hintObject = { text: 'Hint text object', a11yLabel: 'Hint a11y' }
+
+  const descriptionString = 'Description string'
+  const descriptionObject = {
+    text: 'Description text object',
     a11yLabel: 'Description a11y',
   }
-  const error = { text: 'Error text', a11yLabel: 'Error a11y' }
-  const label = { text: 'Label text', a11yLabel: 'Label a11y' }
+
+  const errorString = 'Error string'
+  const errorObject = { text: 'Error text object', a11yLabel: 'Error a11y' }
+
+  const labelString = 'Label string'
+  const labelObject = { text: 'Label text object', a11yLabel: 'Label a11y' }
 
   describe('Header', () => {
-    it('should render text and a11y label', () => {
-      render(<Header text={header} />)
-      expect(screen.root).toHaveTextContent('Header text')
+    it('should render text object and a11y label', () => {
+      render(<Header text={headerObject} />)
+      expect(screen.root).toHaveTextContent('Header text object')
       expect(screen.getByLabelText('Header a11y')).toBeOnTheScreen()
     })
 
+    it('should render simple string', () => {
+      render(<Header text={headerString} />)
+      expect(screen.root).toHaveTextContent('Header string')
+    })
+
+    it('should render text object without a11y label', () => {
+      render(<Header text={{ text: 'Header text object' }} />)
+      expect(screen.root).toHaveTextContent('Header text object')
+      expect(screen.getByLabelText('Header text object')).toBeOnTheScreen()
+    })
+
+    it('should render required text', () => {
```

Comment:
In this test and/or another (the light/dark mode ones for red text?), I think it missed part of the earlier comment: "special required styling/a11y" to validate the red text of "(*Required)" and the a11y being `[text], Required`.